docs/install/install-extensions.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "Installing optional extensions"
+linkTitle: "Install optional extensions"
+weight: 04
+type: "docs"
+showlandingtoc: "false"
+---
+
+To add extra features to your Knative Serving or Eventing installation, you can install extensions
+by applying YAML files using the `kubectl` CLI.
+
+For information about the YAML files in the Knative Serving release, see
+[Installation files](./installation-files.md).
+
+
+# Prerequisites
+
+Before you install any optional extensions, you must install Knative Serving or Eventing.
+See [Installing Serving using YAML files](./install-serving-with-yaml.md)
+and [Installing Eventing using YAML files](./install/install-eventing-with-yaml.md).
```

Comment:
I'd split this into optional stuff for serving and optional stuff for eventing, so that they can have their own prerequisites etc. We only point to the install files for serving above which is confusing.